hyperspy/tests/signal/test_remove_background.py

```diff
@@ -59,3 +60,54 @@ def test_background_remove_pl_int(self):
             background_type='PowerLaw',
             show_progressbar=None)
         assert np.allclose(s1.data, np.zeros(len(s1.data)), atol=60)
+
+
+def test_BackgroundRemoval_gui():
```

Comment:
I find this test a bit too convolved and complicated.

Reply:
You are probably right. It seems you have added a better test module. Should the old one be removed then?

Reply:
Yes, I would suggest be to remove it because it can happen that we spend more time to understand what does the tests than fixing the actual problem!!

Reply:
I have deleted the filed and pushed. The commit shows in my remote repository but not in this PR. I am lost. Any idea what could be happening?

Reply:
OK change pushed